hsm-gateway-applications/scripts/generate-hsm-user.sh

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bash
+set -eo pipefail
+
```

Comment:
Suggest capturing the script directory and using it to qualify all relative path names below:

```bash
SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
```